src/main/java/dk/kb/datahandler/enrichment/FragmentsClient.java

```diff
@@ -0,0 +1,90 @@
+package dk.kb.datahandler.enrichment;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dk.kb.datahandler.config.ServiceConfig;
+import org.apache.http.client.utils.URIBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+/*
+ *  Client for accessing metadata fragments for enrichment (from webservices)
+ */
+public class FragmentsClient {
+    private static final Logger log = LoggerFactory.getLogger(FragmentsClient.class);
+
+    private static FragmentsClient instance;
+
+    private final String baseUrl;
+    private final int maxRetries;
+
+    public static synchronized FragmentsClient getInstance() {
+        if (instance == null) {
+            instance = new FragmentsClient(ServiceConfig.getConfig().getString("fragmentService.baseUrl"),5);
```

Comment:
I believe five retries is a fine number. However, should we consider making this configurable? I don't have a strong opinion on this and one could argue that this is a one time thing, which probably shouldn't be configured anyway. Lets keep it at 5 hardcoded.